samples/graph-activity-feed/csharp/ActivityFeedNotification/Helpers/ChatMembersNotificationRecipient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Graph
```

Comment:
I think these are not the helpers but just a class models.
We can add Models folder and put it there ?

Reply:
Done